### PR TITLE
net-misc/curl: Update to v7.86.0

### PR DESCRIPTION
**Pull request by der-eismann**

### CODE REVIEW

File: net-misc/curl/stone.yml [hunk 1]
```diff
@@ -28,3 +28,7 @@ build       : |
     %make
 install     : |
     %make_install
+packages    :
+    - "%(name)-docs":
```
Comment (ReillyBrogan) [inline net-misc/curl/stone.yml]: Why would the man files for curl be in a -docs subpackage?

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]

File: net-misc/curl/stone.yml (hunk 1 above)
Comment (der-eismann) [reply net-misc/curl/stone.yml]: Aren't man files docs? Or what do you define as docs?

Comment (ReillyBrogan) [reply net-misc/curl/stone.yml]: Docs are usually things like API references. Some packages ship HTML or RTF/Markdown files as documentation that can be referenced on the system. Man files are _technically_ documentation, however we consider `man curl` something essential to the package and so man files are almost never split off to the docs subpackage.

Comment (der-eismann) [reply net-misc/curl/stone.yml]: Got it, thanks for explaining!

**Pull request merged**